modules/installation-network-user-infra.adoc

```diff
@@ -97,6 +103,41 @@ node names. Another supported approach is to always refer to hosts by their
 fully-qualified domain names in both the node objects and all DNS requests.
 endif::azure,gcp[]
 
+ifdef::rhv[]
+.Firewall
+
+Configure your firewall so your cluster has access to required sites.
+
+See also:
+
+ifndef::openshift-origin[]
+* link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/planning_and_prerequisites_guide/index#RHV-manager-firewall-requirements_RHV_planning[Red Hat Virtualization Manager firewall requirements]
+* link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/planning_and_prerequisites_guide#host-firewall-requirements_RHV_planning[Host firewall requirements]
+endif::[]
+ifdef::openshift-origin[]
+* link:https://ovirt.org/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.html#RHV-manager-firewall-requirements_SHE_cli_deploy[oVirt Engine firewall requirements]
+* link:https://ovirt.org/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.html#host-firewall-requirements_SHE_cli_deploy[Host firewall requirements]
+endif::[]
+
+ifeval::["{context}" == "installing-rhv-user-infra"]
+.Load balancers
+
+Configure one or preferably two layer-4 load balancers:
+
+* Provide load balancing for ports `6443` and `22623` on the control plane and bootstrap machines. Port `6443` provides access to the Kubernetes API server and must be reachable both internally and externally. Port `22623` must be accessible to nodes within the cluster.
+
+* Provide load balancing for port `443` and `80` for machines that run the Ingress router, which are usually worker nodes in the default configuration. Both ports must be accessible from within and outside the cluster.
```

Comment:
Can we use `compute nodes` here instead of `worker nodes`?

Reply:
Done.